### ISSUE TITLE
Marker doesn't come on map for lat = 0 and longitude = 0

### ISSUE THREAD

**Issue by riorulzz** [istarkov/google-map-react#163]
Marker is not comng on map if latitude = 0 and longitude=0 :-
<img width="866" alt="screen shot 2016-06-03 at 11 41 11 pm" src="https://cloud.githubusercontent.com/assets/7091402/15788417/16b784f2-29e5-11e6-8322-1de2b79ba03c.png">

It works just fine if latitude = 0.000000000001 and longitude = 0
<img width="904" alt="screen shot 2016-06-03 at 11 41 45 pm" src="https://cloud.githubusercontent.com/assets/7091402/15788439/3088b752-29e5-11e6-8661-d19ecde7f036.png">

Is this expected behaviour ?

**Comment by istarkov** [comment]
do you really need this position :-) ?

**Comment by riorulzz** [comment]
Yes .. I need to plot this point on map.. can you please fix this .. ?

**Comment by istarkov** [comment]
for now u could use 0.000000000001 lat long coords, as it wil give subpixel accuracy. I have no time now to resolve edgecases.
